doc/varnish-configuration.rst

```diff
@@ -4,7 +4,7 @@ Varnish Configuration
 ---------------------
 
 Below you will find detailed Varnish configuration recommendations for the
-features provided by this library. The examples are tested with Varnish
+features provided by this library. These examples have been tested with Varnish
```

Comment:
actually its not examples, but reference documentation. lets say:

... features provided by this library. The configuration is provided for Varnish 3 and 4.